worker/jobTypes/S3Publish.js

```diff
@@ -0,0 +1,52 @@
+const fs = require('fs-extra');
+const workerUtils = require('../utils/utils');
+
+class S3PublishClass {
+
+  constructor(currentJob) {
+    this.currentJob = currentJob;
+  }
+
```

Comment:
I would put a placeholder in for the productionPush even though we aren't ready to code it yet.